client/verta/tests/unit_tests/deployment/test_build_scan.py

```diff
@@ -50,3 +50,23 @@ def test_get_scan(mock_conn, mocked_responses, build_dict, build_scan_dict):
         build_scan = build.get_scan()
 
     assert_build_scan_fields(build_scan, build_scan_dict)
+
+@settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+@given(build_dict=build_dict(), build_scan_dict=build_scan_dict())
+def test_start_scan(mock_conn, mocked_responses, build_dict, build_scan_dict):
+    """Verify we can construct a BuildScan object from start_scan."""
+    build_dict["creator_request"]["scan_external"] = True
+    build_scan_dict["creator_request"]["scan_external"] = True
```

Comment:
We should add a parameter to the strategies to optionally force `scan_external` instead of overriding the item here. This is going to throw off `hypothesis`'s example shrinking behavior, and also breaks the `build_scan_dict` strategy because it has a check for `scan_external`.

Reply:
and Done